src/MPAGSCipher/TransformChar.hpp

```diff
@@ -3,7 +3,6 @@
 
 #include <string>
 
-//! Transliterate char to string
-std::string transformChar(const char inputChar);
+std::string transformChar(const char in);
 
-#endif    // MPAGSCIPHER_TRANSFORMCHAR_HPP
\ No newline at end of file
+#endif  
```

Comment:
Why did you remove the comment?
While it is not needed for such a short file, it is a convention we are trying to keep in this project